blueocean-rest-impl/pom.xml

```diff
@@ -51,9 +51,9 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>ssh-credentials</artifactId>
         </dependency>
-        <dependency>
-            <groupId>com.jcraft</groupId>
-            <artifactId>jsch</artifactId>
+        <dependency> <!-- need this to solve a JSch linkage problem -->
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>git-client</artifactId>
```

Comment:
Dependence of git-client is 🐛 Do not see master failing in CI run or locally with any linkage error. Perhaps something local in your env? Need to get more details before we know whats the right dependency we need to add if/any.

Reply:
Well, we need `jsch`. It doesn't work to include it directly. Master isn't failing because there are no `git` tests in ATH. Again, it only fails when you try to load something in the editor using a Git pipeline.

Reply:
@kzantow ah ok, this is good info, I can try to reproduce it. will get back.